messaging_components/services/__init__.py

```diff
@@ -39,6 +41,10 @@ def create_service(executor: Executor, service_name: str=None, **kwargs) -> Serv
             container_name = None
             if isinstance(executor, ExecutorContainer):
                 container_name = executor.container_name
+            elif isinstance(executor, ExecutorAnsible):
```

Comment:
@michalxo This is a less restrictive condition, and so it should be the last one. If you put it here, then the other one (last elif below) will never get executed.